src/sentry/integrations/gitlab/client.py

```diff
@@ -418,3 +418,21 @@ def get_pr_diffs(self, repo: Repository, pr: PullRequest) -> list[dict[str, Any]
         project_id = repo.config["project_id"]
         path = GitLabApiClientPath.build_pr_diffs(project=project_id, pr_key=pr.key, unidiff=True)
         return self.get(path)
+
+    def create_status(self, repo: str, sha: str, data: dict[str, Any]) -> Any:
```

Comment:
Felt a bit hacky naming this `repo` but being consistent with the existing code for now.